.github/workflows/ci.yml

```diff
@@ -12,23 +12,95 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  main:
-    name: Build and Deploy
+  build:
+    name: Build
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
       - name: Use Node.js
         uses: actions/setup-node@v3
         with:
           node-version: '16.x'
           cache: 'npm'
-      - uses: nrwl/nx-set-shas@v3
+      - name: Use nx-set-sha
+        uses: nrwl/nx-set-shas@v3
       - name: Install
         run: npm ci
       - name: Package
         run: npm run package
+      - name: Store build
+        uses: actions/upload-artifact@v3
+        with:
+          name: storybook-static
+          path: apps/storybook/storybook-static
+  test:
+    name: Test
+    runs-on: ubuntu-20.04
```

Comment:
What is this fixed to 20.04 and the build is latest?

Reply:
This effects the browser version number so it's good to fix it for the test suite.
We could fix the build as well - but generally the build context for github actions is set to latest. I'm happy to change if you want.